### PR TITLE
[Portfolio] add current reference market value to portfolio_pretty_print

### PR DESCRIPTION
**Pull request by techfreaque**
see pull request in octobot services here https://github.com/Drakkar-Software/OctoBot-Services/pull/136

### CODE REVIEW

File: octobot_commons/pretty_printer.py [hunk 1]
```diff
@@ -150,8 +153,20 @@ def global_portfolio_pretty_print(
...
+                        f" ({get_min_string_from_number(currency_values[currency]*asset_dict[constants.PORTFOLIO_TOTAL])} {ref_market_name})"
+                        if ref_market_name != currency else ""
+                    )
+                    if markdown:
+                        ref_value = "{:<12}".format(ref_value)
+                except KeyError:
+                    # no currency value
+                    pass
+            
+            holding_str = f"{total} {available} {currency}{ref_value}"
```
Comment (GuillaumeDSM) [inline octobot_commons/pretty_printer.py:169]: I'm wondering if printing the available amount is really necessary, maybe we should use the value in ref market instead and display the total locked funds in another command. What is you opinion on this ? 
I think we should not display all 3 at the same time because it makes it on multiple lines on telegram mobile which makes it harder to read

Comment (techfreaque) [reply octobot_commons/pretty_printer.py:169]: ah yeah that makes sense, I'll swap it and we add the command later

**Comment by techfreaque** [comment]
regarding your questions:

- [Profile] allow parameters in profile import url
        why replacing space in file name ?, can it break the uniqueness check?
        >yes, if you import a profile with spaces multiple times, it will create a new one each time

- [Portfolio] add current reference market value to portfolio_pretty_print
        should we remove comments ?
        >I removed them

- [Databases] add optimizer schedule config table
        not sure it's necessary
        >its not fully implemented yet, but its supposed to be the run queue for the genetic optimizer

- [Logger] add DebuggingReporter
1. please add drakkar software copyright
I added it
2. could we add tests plz ?
I'll add them in the next few days
3. file needs to be added to setup.py to be cythonized
its added

**Comment by techfreaque** [comment]
I dont know why it fails to cythonize, do you know why?

File: octobot_commons/profiles/profile_sharing.py [hunk 2]
```diff
@@ -238,7 +238,7 @@ def _get_unique_profile_folder_from_name(profile) -> str:
     folder = _get_unique_profile_folder(
         os.path.join(os.path.split(profile.path)[0], profile.name)
     )
-    return os.path.split(folder)[1]
+    return os.path.split(folder)[1].replace(" ", "_")
```
Comment (Herklos) [inline octobot_commons/profiles/profile_sharing.py:241]: Why do we need to replace spaces here?

File: octobot_commons/display/display_translator.py [hunk 3]
```diff
@@ -367,6 +367,8 @@ def _get_element_schema_type(self, options):
                 return default_type
         except IndexError:
             pass
+        except TypeError as error:
+            raise TypeError("a user input element is malformed.") from error
```
Comment (Herklos) [inline octobot_commons/display/display_translator.py:371]: 👍

File: octobot_commons/profiles/profile_sharing.py (hunk 2 above)
Comment (techfreaque) [reply octobot_commons/profiles/profile_sharing.py]: I removed it, I think its not required

File: octobot_commons/pretty_printer.py [hunk 4]
```diff
@@ -186,11 +210,10 @@ def pretty_print_dict(dict_content, default="0", markdown=False) -> str:
...
     if dict_content:
         result_str = octobot_commons.DICT_BULLET_TOKEN_STR
-        return (
-            f"{result_str}{code}"
-            f"{octobot_commons.DICT_BULLET_TOKEN_STR.join(f'{value} {key}' for key, value in dict_content.items())}"
-            f"{code}"
+        pretty_dict = octobot_commons.DICT_BULLET_TOKEN_STR.join(
+            f"{value} {key}" for key, value in dict_content.items()
         )
+        return f"{result_str}{code}" f"{pretty_dict}" f"{code}"
```
Comment (Herklos) [inline octobot_commons/pretty_printer.py:216]: Why this change?

Comment (techfreaque) [reply octobot_commons/pretty_printer.py]: Oh this slipped in for debugging I think, I removed it

**Review by Herklos — APPROVED** [review]
